pkg/cli/debug_job_trace_test.go

```diff
@@ -120,64 +117,38 @@ func TestDebugJobTrace(t *testing.T) {
 	// Wait for the job to record information in the trace span.
 	<-recordedSpanCh
 
-	tempDir, cleanup := testutils.TempDir(t)
-	defer cleanup()
-	targetFilePath := filepath.Join(tempDir, "tracetest")
-	args := []string{strconv.Itoa(int(id)), targetFilePath}
+	args := []string{strconv.Itoa(int(id))}
 	pgURL, _ := sqlutils.PGUrl(t, c.TestServer.ServingSQLAddr(),
 		"TestDebugJobTrace", url.User(security.RootUser))
 
-	_, err := c.RunWithCaptureArgs([]string{`debug`, `job-trace`, args[0], args[1], fmt.Sprintf(`--url=%s`, pgURL.String()), `--format=csv`})
+	_, err := c.RunWithCaptureArgs([]string{`debug`, `job-trace`, args[0], fmt.Sprintf(`--url=%s`, pgURL.String()), `--format=csv`})
 	require.NoError(t, err)
-	actual, err := ioutil.ReadFile(targetFilePath)
-	if err != nil {
-		t.Errorf("Failed to read actual result from %v: %v", targetFilePath, err)
-	}
-
-	if err := matchCSVHeader(string(actual), jobTraceHeader); err != nil {
-		t.Fatal(err)
-	}
-
-	operationName := fmt.Sprintf("BACKUP-%d", id)
-	exp := [][]string{
-		// This is the span recording we injected above.
-		{`\d+`, `\d+`, operationName, ".*", ".*", "server.serverpb.TableStatsRequest", "{\"@type\": \"type.googleapis.com/cockroach.server.serverpb.TableStatsRequest\", \"database\": \"foo\", \"table\": \"bar\"}"},
-	}
-	if err := MatchCSV(string(actual), exp); err != nil {
-		t.Fatal(err)
-	}
+	checkBundle(t, id, "node1-trace.txt node1-trace.json node1-jaeger.json")
```

Comment:
It looks like this supports being called as `checkBundle(t, id, "node1-trace.txt", "node1-trace.json", "node1-jaeger.json")` -- was that the intention?

Reply:
Copy pasta from explain_bundle_test, fixed.